client/src/_shared/_hooks/useAuthentication.js

```diff
@@ -1,24 +1,18 @@
 import dayjs from 'dayjs'
+import { useSelector } from 'react-redux'
 
+// TODO: Something isn't working here, if I hit an authenticated route directly, it logs me out
```

Comment:
cc @arku @cjhaddad if you have a minute to look at this and see what I'm missing with grabbing the token from state when hitting an AuthenticatedRoute directly (i.e., login, land on getting-started, and then go to the browser bar and hit enter to hard reload getting-started and you'll be logged out)